start_cloudserver.sh

```diff
@@ -0,0 +1,2 @@
+
+docker run --name s3server -p 8000:8000 -e REMOTE_MANAGEMENT_DISABLE=1 -e SCALITY_ACCESS_KEY_ID=accessKey1 -e SCALITY_SECRET_ACCESS_KEY=verySecretKey1 zenko/cloudserver
```

Comment:
Port 8000 looks weird, because our (default) config is assuming 8003 now

Reply:
Thanks for pointing at the scripts. Will delete them since they are not needed anylonger.

Reply:
Fixed